CMakeLists.txt

```diff
@@ -158,10 +158,10 @@ endif()
 
 # Set custom build flags for RelWithDebInfo.
 # -DNDEBUG purposely omitted because we want assertions.
-if(HAS_OG_FLAG)
+if(NOT MSVC AND HAS_OG_FLAG)
```

Comment:
HAS_OG_FLAG will always be false here because of line 156, so I'll leave this out.